test/src/main/java/org/apache/accumulo/test/functional/ManyWriteAheadLogsIT.java

```diff
@@ -213,7 +213,8 @@ private void addOpenWals(ServerContext c, Set<String> allWalsSeen) throws Except
           allWalsSeen.add(entry.getKey());
           foundWal = true;
         } else {
-          log.debug("The WalState for {} is {}", entry.getKey(), entry.getValue()); // CLOSED or UNREFERENCED
+          log.debug("The WalState for {} is {}", entry.getKey(), entry.getValue()); // CLOSED or
+                                                                                    // UNREFERENCED
```

Comment:
This seems unrelated. If having trouble with the formatter, could just remove that comment at the end of the line, or put it above the log statement.

Reply:
Disregard this one. I fixed the formatting in the master branch.